src/deliver-mobile/app-shield/intro.md

```diff
@@ -134,7 +134,7 @@ Protections that are available for iOS builds.
 * Code injection protection
 * Debugger protection
 * Screen mirroring detection
-* Screenshot protection ([see limitation](#general))
+* Screenshot protection (Only blocks system screenshots[see limitation](#general))
```

Comment:
Can you please define what is meant by "system screenshots"?

Reply:
System screenshots would be screenshots made by external programs, user-initiated are the ones made by the user with the device's built-in functionality